acceptancetests/jujupy/client.py

```diff
@@ -1330,6 +1333,17 @@ def juju(self, command, args, check=True, include_e=True,
             command, args, self.used_feature_flags, self.env.juju_home,
             model, check, timeout, extra_env, suppress_err=suppress_err)
 
+    def reboot(self, machine):
```

Comment:
I made a standalone method for rebooting machines and also plan on back-porting it to 2.9 so we can use the same API instead of `juju ssh` (develop) and `juju run` (2.9)